src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -730,3 +730,62 @@ def computer_config_show(computer, user, defaults, as_option_string):
             else:
                 table.append((f'* {name}', '-'))
         echo_tabulate(table, tablefmt='plain')
+
+
+@verdi_computer.group('export', cls=LazyConfigureGroup)
+def computer_export():
+    """Exports the Authinfo details for a computer (and user)."""
+
+
+@computer_export.command('setup')
+@arguments.COMPUTER()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False))
+@with_dbenv()
+def computer_export_setup(computer, output_file):
+    """Export computer setup to a yaml file."""
+    import yaml
+
+    computer_setup_data = {
+        'label': computer.label,
+        'description': computer.description,
+        'hostname': computer.hostname,
+        'transport': computer.transport_type,
+        'scheduler': computer.scheduler_type,
+        'work_dir': computer.get_workdir(),
+        'shebang': computer.get_shebang(),
+        'mpirun_command': ' '.join(computer.get_mpirun_command()),
+        'mpiprocs_per_machine': computer.get_default_mpiprocs_per_machine(),
+        'default_memory_per_machine': computer.get_default_memory_per_machine(),
+        'prepend_text': computer.get_prepend_text(),
+        'append_text': computer.get_append_text(),
+        'use_double_quotes': computer.get_use_double_quotes(),
+    }
+    try:
+        with open(output_file, 'w', encoding='utf-8') as yfhandle:
+            yaml.dump(computer_setup_data, yfhandle)
+        echo.echo_success(f"Computer<{computer.pk}> {computer.label} setup exported to file '{output_file}'.")
+    except Exception:
```

Comment:
Can we make this more explicit? Do we know which exceptions to expect here? `FileExistsError` should be captured already in the `@arguments.OUTPUT_FILE`. Anything else? Possibly also communicate the failure via `echo.echo_error`/`echo.echo_critical` rather than just raising the Exception.

Reply:
Ah, seems like I misunderstood the behavior of `exists=False` in
```python
@arguments.OUTPUT_FILE(type=click.Path(exists=False))
```
Thought that would raise if the file exists already, but it just means it doesn't necessarily need to exist ^^

I think exporting configuration files, it's fine to overwrite by default, as accidentally overwriting is not too critical. In my dumping PR, I added it as a `click` option with `default=False`, as there I'm really doing a recursive deletion of files and directories, so one might lose more relevant data.

Reply:
You think it is worth to make an issue with priority nice-to-have for a  `--no-overwrite` feature for the export commands?